src/ui/public/vis/map/kibana_map.js

```diff
@@ -95,6 +102,85 @@ function makeLegendControl(container, kibanaMap, position) {
   return new LegendControl(container, kibanaMap, position);
 }
 
+const makeZoomWarningMsg = (function () {
+  let disableZoomMsg = false;
+  const setZoomMsg = boolDisableMsg => disableZoomMsg = boolDisableMsg;
+
+  class ZoomWarning extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        disabled: false
+      };
+    }
+
+    render() {
+      return (
+        <div>
+          <p>
+            <FormattedMessage
+              id="common.ui.vis.kibanaMap.zoomWarning"
+              defaultMessage={`You've reached the maximum number of zoom
+              levels. To zoom all the way in, upgrade to the
+              {defaultDistribution} of Elasticsearch and Kibana. You'll get
+              access to additional zoom levels for free through the {ems}.
+              Or, you can configure a custom WMS compliant server.`}
+              values={{
+                defaultDistribution: (
+                  <a
+                    target="_blank"
+                    href="https://www.elastic.co/downloads/kibana"
+                  >
+                    {`default distribution `}
+                  </a>
+                ),
+                ems: (
+                  <a
+                    target="_blank"
+                    href="https://www.elastic.co/elastic-maps-service"
+                  >
+                    {`Elastic Maps Service`}
+                  </a>
+                )
+              }}
+            />
+          </p>
+          <EuiSpacer size="xs"/>
+          <EuiButtonEmpty
+            size="s"
```

Comment:
You can add `flush="left"` to align this better withe the left margin of the text that's above it.

Reply:
Done!